M2/Macaulay2/m2/quotring.m2

```diff
@@ -281,7 +281,7 @@ char QuotientRing := (stashValue symbol char) ((S) -> (
      g := generators gb relns;
      if g == 0 then return char ring g;
      m := g_(0,0);
-     if liftable(m,ZZ) then lift(m,ZZ) else 0))
+     lift(m,ZZ,Verify=>false) ?? 0))
```

Comment:
Not sure if I understand the motivation in this change. I like `??` and `??=` in certain situations (e.g. caching) but I'm concerned that in places like this it might make debugging more difficult. For instance, I imagine we _would_ like to see an error if `isLiftable` returned true but `lift` failed here, because that would be a bug and an error would help us locate it, but `??` hides the error here and silently returns zero.

Reply:
in general I would agree with you but here's a pretty clear-cut case where we're basically calling `lift` twice for no reason.

Reply:
I'm actually a little confused about what's happening in this code, but sure.